directory/directory_test.go

```diff
@@ -58,7 +58,7 @@ func TestGetPutManifest(t *testing.T) {
 func TestGetPutBlob(t *testing.T) {
 	ref, tmpDir := refToTempDir(t)
 	defer os.RemoveAll(tmpDir)
-	cache := blobinfocache.NewMemoryCache()
+	cache := memory.New()
```

Comment:
(Absolutely non-blocking: `memory.New()` can be anything, and `boltdb.New()` is even a bit confusing. If we are retaining `NoCache`, maybe `NewCache` would still be worth it?  )

Reply:
> Absolutely non-blocking: memory.New() can be anything...

But the import path is `github.com/containers/image/pkg/blobinfocache/memory`, which includes "cache".  And the receiving `cache` here also reminds readers about what this is.  Folks who want to include more information are free to name their import:

```go
import (
  blobinfomemorycache "github.com/containers/image/pkg/blobinfocache/memory"
)
...
cache := blobinfomemorycache.New()
```

So I like `New`.  But I'm not a maintainer; if you want me to make it `NewCache` or whatever, just tell me to change it ;).

Reply:
I did say that it is absolutely non-blocking :)

Just something to consider, and somewhat a counterweight to Valentin’s naming concerns.